jboss-image-streams.json

```diff
@@ -61,12 +63,13 @@
                         "name": "1.3",
                         "annotations": {
                             "description": "JBoss Web Server 3.0 Tomcat 7 S2I images.",
-                            "iconClass": "icon-jboss",
-                            "tags": "hidden,builder,tomcat,tomcat7,java,jboss,xpaas",
-                            "supports":"tomcat7:3.0,tomcat:7,java:8,xpaas:1.3",
+                            "iconClass": "icon-tomcat",
+                            "tags": "builder,tomcat,tomcat7,java,jboss,hidden",
+                            "supports": "tomcat7:3.0,tomcat:7,java:8:1.3",
                             "sampleRepo": "https://github.com/jboss-openshift/openshift-quickstarts.git",
                             "sampleContextDir": "tomcat-websocket-chat",
-                            "version": "1.3"
+                            "version": "1.3",
+                            "openshift.io/display-name": "Red Hat JBoss Web Server 3.0 Tomcat 7"
```

Comment:
We've had a request to add "Apache" before Tomcat.

Reply:
ok, adding